crates/model/src/orders/limit_if_touched.rs

```diff
@@ -176,7 +247,7 @@ impl Order for LimitIfTouchedOrder {
         self.instrument_id
     }
 
-    fn symbol(&self) -> Symbol {
+    fn symbol(&self) -> crate::identifiers::Symbol {
```

Comment:
We don't need to fully qualify identifiers.

Reply:
reverted, thx